src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App'
+import './index.css'
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
```

Comment:
In case you remove The React import as suggested above, then for the StrictMode, you can import it directly from React like this,
```suggestion
import {StrictMode} from "React";
  <StrictMode>
    <App />
  </StrictMode>,
```